recipes/django-libsass/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "django-libsass" %}
+{% set version = "0.7" %}
+{% set sha256 = "49db3334b87e1f7955c4f9fb9945bc296f8bfd27a14d6d89706e4b0e5dc5de1c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
+    - django-compressor >=1.3
+    - libsass >=0.7.0
+
+test:
+  import:
+    - django_libsass
+about:
+  home: https://github.com/torchbox/django-libsass
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  summary: 'A django-compressor filter to compile SASS files using libsass'
+  description: |
+    A django-compressor filter to compile Sass files using libsass.
+  doc_url: https://github.com/torchbox/django-libsass
+  dev_url: https://github.com/torchbox/django-libsass
+
+extra:
+  recipe-maintainers:
+    - cshaley
```

Comment:
@cshaley, you happy to be a maintainer here?

Reply:
Yes